apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -306,6 +306,11 @@ a crash dump
           #{ mapping => "emqx_machine.backtrace_depth"
            , default => 23
            })}
+    , {"applications",
+       sc(emqx_schema:comma_separated_atoms(),
+          #{ mapping => "emqx_machine.applications"
+           , default => [gproc, esockd, ranch, cowboy, emqx]
```

Comment:
we can probably hide the defaults, only make additions configurable.

Reply:
✅